doc/srv6/srv6_hld.md

```diff
@@ -0,0 +1,598 @@
+# Segment Routing over IPv6 (SRv6) HLD  
+
```

Comment:
Please check LOCAL_SID to MY_SID replacements in the diagram as well, I see LOCAL_SID usage in the diagrams, please correct them.